db/migrate/20221117072604_add_devise_to_users.rb

```diff
@@ -48,4 +48,4 @@ def self.down
     # model already existed. Please edit below which fields you would like to remove in this migration.
     raise ActiveRecord::IrreversibleMigration
   end
-end
\ No newline at end of file
+end
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you fix the migration error as right now it raises the below error when trying to run the migrations i.e `rails db:migrate`. Make sure you fix this00 before requesting a new review. So, impossible to check whether your project is it is running or not.
![Screenshot from 2022-11-17 17-53-46](https://user-images.githubusercontent.com/83042742/202497533-2452ac15-ca17-4736-8a47-1d0433e09277.png)

Reply:
Thank you for your answer, I have checked out everything but I don't see a problem with my database, I don't have this problem when I ran migrate, I don't know if you should have run rails db:create instead of migrate and create the database, or if you did something differently.
![image](https://user-images.githubusercontent.com/98361289/202507432-087c494d-5ef6-4f0e-bb49-7735d330bbaa.png)

Reply:
Hi @renercrows 

This is what I did. I even dropped and created 5 times the last command for running the migration blocks. 
`rails db:drop db:create db:migrate`

Are you sure to have pushed all your changes on GitHub?

Reply:
Hi @renercrows 

I just tried it on another computer but still facing the same result:
![Screenshot from 2022-11-17 19-37-18](https://user-images.githubusercontent.com/83042742/202517997-8dec15e4-b9de-4ce7-a341-cbf095cb0442.png)

Reply:
I did, I don't know what else to do, cause I pushed everything, I will push again but is just because I did migration several times, even deletes likes_counters and migrated again to created again.

Reply:
Hi @renercrows 

I think it is normal on your side to not see this error because you running the existing migrations.
But if you try to drop the old one, create a new one then run migrate, I think you can also see that error.
i.e `rails db:drop db:create db:migrate`